alpha_rectification: -1.0
baseline_mm: 494.2638645727461
calibration_date: '2025-06-16T08:54:55'
calibration_flags: 16644
disparity_to_depth_Q:
- - 1.0
  - 0.0
  - 0.0
  - -759.5778198242188
- - 0.0
  - 1.0
  - 0.0
  - -1161.2067947387695
- - 0.0
  - 0.0
  - 0.0
  - 1293.932015967729
- - 0.0
  - 0.0
  - -2.023210822552089
  - 0.0
essential_matrix:
- - -0.011357028220501485
  - 0.29998453167854455
  - 0.3925578316371129
- - 0.06075212365645826
  - 0.00820625995198065
  - 0.006335370754198608
- - -0.48990640900090443
  - -0.023260951831524074
  - 0.004951920305370321
fundamental_matrix:
- - -4.292188273088115e-08
  - 1.1337385660926218e-06
  - 0.0013115951594249993
- - 2.296019237257254e-07
  - 3.1014110423905134e-08
  - -0.00016328201684403818
- - -0.0024457450144084805
  - -0.0009879849443077094
  - 1.0
image_size:
- 1600
- 1200
projection_P1:
- - 1293.932015967729
  - 0.0
  - 759.5778198242188
  - 0.0
- - 0.0
  - 1293.932015967729
  - 1161.2067947387695
  - 0.0
- - 0.0
  - 0.0
  - 1.0
  - 0.0
projection_P2:
- - 1293.932015967729
  - 0.0
  - 759.5778198242188
  - 0.0
- - 0.0
  - 1293.932015967729
  - 1161.2067947387695
  - 639.543838706614
- - 0.0
  - 0.0
  - 1.0
  - 0.0
rectification_R1:
- - 0.9988697789857214
  - 0.046002186430967325
  - -0.011956733357689146
- - -0.043800323927016604
  - 0.7931854344511359
  - -0.607402994887621
- - -0.018457959066613183
  - 0.6072402040528387
  - 0.7943038702719267
rectification_R2:
- - 0.9998353838658322
  - 0.017994478057679508
  - -0.0023246353038315027
- - -0.01756996471302375
  - 0.9921955988597099
  - 0.12344711395332289
- - 0.004527859300732446
  - -0.12338594880639078
  - 0.99234842980039
rms_error: 5.025192528789068
roi1: !!python/tuple
- 97
- 0
- 1427
- 806
roi2: !!python/tuple
- 35
- 801
- 1510
- 399
roi_validity:
  roi1_valid: true
  roi2_valid: true
rotation_matrix:
- - 0.9993913440083518
  - 0.034807871840382254
  - 0.0023137802669124397
- - -0.023206895516114195
  - 0.7128979737459351
  - -0.7008836701118412
- - -0.026045758228244932
  - 0.7004033774096777
  - 0.7132718467677189
translation_analysis:
  baseline_mm: 494.2638645727461
  euler_x_deg: 44.47845968685765
  euler_y_deg: 1.4924807983746002
  euler_z_deg: -1.3302279062580697
  tx_mm: -8.684198659465913
  ty_mm: 490.4064311044704
  tz_mm: 61.015447612921456
translation_vector:
- - -0.008684198659465913
- - 0.4904064311044704
- - 0.06101544761292146
valid_pairs: 12
